### ISSUE TITLE
[Enhancement] Basic Line type Chart Component

### ISSUE THREAD

**Issue by MadsBuchmann** [kirbydesign/designsystem#1727]
- [X] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
The [new chart component](https://cookbook.kirby.design/home/showcase/chart) should be extended to support line charts.

## Describe the solution you'd like
A screenshot of the desired line chart has been attached. It should simply be implemented as a new 'type' for the chart component, that displays the supplied data as a line. It should also be possible to have several lines by supplying several datasets to the chart.

The marked area in the background is not needed for now. It should just be possible to draw 3 separate lines with 3 different colours. 

The implementation of the labels are handled in this issue: #1729 

![image](https://user-images.githubusercontent.com/17269698/131500955-9aa8fbf2-8a76-40e3-b8b5-1302ba71e79f.png)

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](../CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [X] Request that the issue is [UX refined](../CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [X] Request that the issue is [tech refined](../CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [X] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [X] Signal to others you are working on the issue by assigning yourself.
- [X] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".
